_posts/2020-04-07-charity-shop-exchange.md

```diff
@@ -0,0 +1,63 @@
+---
+author: John
+title: Charity Shop Exchange
+---
+
+# Charity Shop Exchange
+
+<span class="image right">
+    <img src="/images/blog/charity-shop-exchange.png" alt="Charity Shop Exchange">
+</span>
```

Comment:
This suffers from the same shortcomings I mentioned in #64.  Still, not a blocker.

Reply:
Unfortunately, there no way to do this that I know of.

HTML is valid markdown though so not too concerned.